From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Mon, 3 Jan 2022 05:13:59 +0900
Subject: test-repart: append /sbin and /usr/sbin to $PATH= to make sfdisk can
 be found

Fixes #21972.

(cherry picked from commit 329a5b91d9ec780b2ce84ee336448291a568ae0d)
---
 src/partition/test-repart.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/partition/test-repart.sh b/src/partition/test-repart.sh
index 525be8e..cf4a634 100755
--- a/src/partition/test-repart.sh
+++ b/src/partition/test-repart.sh
@@ -8,6 +8,8 @@ set -o pipefail
 repart="${1:?}"
 test -x "$repart"
 
+PATH=$PATH:/sbin:/usr/sbin
+
 D="$(mktemp --tmpdir --directory "test-repart.XXXXXXXXXX")"
 
 # shellcheck disable=SC2064
